crates/evm/core/src/opts.rs

```diff
@@ -102,7 +102,6 @@ impl EvmOpts {
     pub fn local_evm_env(&self) -> revm::primitives::Env {
         let mut cfg = CfgEnv::default();
         cfg.chain_id = self.env.chain_id.unwrap_or(foundry_common::DEV_CHAIN_ID);
-        cfg.spec_id = SpecId::MERGE;
```

Comment:
is this right? Don't we override it on purpose?

Reply:
this no longer exists unfortunatelly...